profiles/models_test.py

```diff
@@ -0,0 +1,18 @@
+"""
+Model tests
+"""
+# pylint: disable=no-self-use
+from django.test import TestCase
+from profiles.factories import ProfileFactory, UserFactory
+
+
+class ProfileTests(TestCase):
+    """tests for the profile model"""
+
+    def student_id_on_save(self):
+        """test that a student id is set on save"""
+        user = UserFactory.create()
+        profile = ProfileFactory.create(user=user)
```

Comment:
`ProfileFactory.create` should have saved the `Profile` to the database so it's surprising that `student_id` is `None` here. Maybe `ProfileFactory.create` is not calling save? Can you rewrite the test to use `Profile.objects.create` instead?